cmd/thin_entrypoint/main.go

```diff
@@ -153,12 +153,19 @@ func getFileAndHash(filepath string) ([]byte, []byte, error) {
 	return content, hash.Sum(nil), nil
 }
 
-func (o *Options) createKubeConfig(prevCAHash, prevSATokenHash []byte) ([]byte, []byte, error) {
-	caFileByte, caHash, err := getFileAndHash(serviceAccountCAFile)
+func (o *Options) createKubeConfig(caFilePath, saTokenFilePath string, prevCAHash, prevSATokenHash []byte) ([]byte, []byte, error) {
```

Comment:
are these mostly unused params necessary? looks like they're for testing, is there another way to handle it for the tests? thanks